src/ontology/mondo-edit.obo

```diff
@@ -560078,6 +560087,7 @@ is_a: MONDO:0020573 {source="OMIM:607154"} ! inherited disease susceptibility
 intersection_of: MONDO:0020573 ! inherited disease susceptibility
 intersection_of: has_material_basis_in_germline_mutation_in http://identifiers.org/hgnc/5973 ! IL13
 intersection_of: predisposes_towards MONDO:0011786 ! allergic rhinitis
+relationship: has_material_basis_in_germline_mutation_in http://identifiers.org/hgnc/5973 ! IL13
```

Comment:
mondo.owl v2024-12-03 does not have any source provenance for this gene association